### PR TITLE
Add contribution, CoC and Security doc

### PR DESCRIPTION
**Pull request by Emily-Jiang**
We need to add the contribution doc in this repo to encourage others to contribute.

### CODE REVIEW

**Comment by Emily-Jiang** [comment]
> @Emily-Jiang thanks for creating these additions!
> 
> I would suggest to add only Asciidoc formated files here and not mix it up with Markdown (of any flavor).
> 
> 2nd important point is, what are the supported versions? I asked for that in one of our last meetings and I did not got an answer, which looks for me like something "only the last version is supported yet", as this is the minimum requirement. But vendors might think about it differently... So I think we need to find the answer as soon as possible to finish the Security.adoc part.

I am not sure about which version part. It is best to reach out to emo in Eclipse Foundation. I was told to put these files in place.

**Comment by Emily-Jiang** [comment]
I have now converted all .md to .adoc.

File: CONTRIBUTING.adoc [hunk 1]
```diff
@@ -0,0 +1,32 @@
+//
+// Copyright (c) 2018 Contributors to the Eclipse Foundation
```
Comment (Azquelt) [inline CONTRIBUTING.adoc:2]: Should the copyright date be this year?

File: .github/CODE_OF_CONDUCT.adoc [hunk 2]
```diff
@@ -0,0 +1,47 @@
+= Community Code of Conduct
+
+**Version 1.2
+August 19, 2020**
```
Comment (Azquelt) [inline .github/CODE_OF_CONDUCT.adoc:4]: There is an updated version of the [Community Code of Conduct][coc-html] (and a [markdown version][coc-markdown]).

While it's useful to be consistent in using asciidoc, if we're including standard documents from the eclipse foundation which are only for display within github, I think it's ok to keep them in the original format to make it easier to keep them updated.

[coc-html]: https://www.eclipse.org/org/documents/Community_Code_of_Conduct.php
[coc-markdown]: https://raw.githubusercontent.com/eclipse/.github/master/CODE_OF_CONDUCT.md

File: CONTRIBUTING.adoc [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+Do you want to contribute to this project? Here is what you can do:
+
+* Fork the repository, make changes, then do a pull request.
+** Before you open a pull request, please make sure you have signed the https://www.eclipse.org/legal/ECA.php[Eclipse Contributor Agreement]
+* https://github.com/eclipse/microprofile-telemetry/issues[Create or fix an issue].
+* Join our https://calendar.google.com/calendar/embed?src=gbnbc373ga40n0tvbl88nkc3r4%40group.calendar.google.com[weekly meeting]. 
+** https://docs.google.com/document/d/1lNeJbn6zaQNZ0adoko_Omt7MQW6en4Mh[Minutes and Agenda].
+* Join the discussions on the https://groups.google.com/forum/#!forum/microprofile[MicroProfile Google Group]
+* https://microprofile.io/blog/[Contribute a blog post].
```
Comment (Azquelt) [inline CONTRIBUTING.adoc:30]: I don't see anything on this page about how to contribute to the blog.

**Comment by Emily-Jiang** [comment]
I removed the CoC and Security as MicroProfile community would like to find out what should be adopted by MicroProfile community. These files will be added via a separate PR after some agreement has been reached.

**Review by Azquelt — APPROVED** [review]

**Pull request merged**